### PR TITLE
New 404 page

### PR DESCRIPTION
**Pull request by mrsellars**

### CODE REVIEW

**Comment by GBKS** [comment]
How did you generate the HTML in 404.html? Shouldn't that also be a markdown template that gets compiled from the various includes for header, footer, etc? I'm curious if that's some Jekyll thing that I am not aware of.

**Comment by mrsellars** [comment]
Good catch! It was a local testing file. You are right, content should be in Markdown. I think it was too late for a commit :). Will fix that in the batch of changes.

**Comment by mrsellars** [comment]
Getting there. Desktop version is ready. Next stop, mobile.

404--desktop

**Comment by mrsellars** [comment]
The Mobile version is ready. I changed it from draft to ready for review.

File: 404.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+permalink: /404.html
+layout: error
```
Comment (ConorOkus) [inline 404.md:3]: I think we should call this `page-not-found` to be a bit clearer and differentiate from other types of errors

File: _layouts/error.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
```
Comment (ConorOkus) [inline _layouts/error.html:1]: Similar to comment to above - `page-not-found`

**Comment by ConorOkus** [comment]
Good work so far, I tested this on mobile and looks really good. I would just make the change Christoph described and keep the content separate from the markup or just include everything in the layout directly which might be preferable actually

**Comment by GBKS** [comment]
Nice work. Looks really good on desktop and mobile. Just had a little bit of feedback on spacing, see the image below.

For the spacing between main copy and button, I would only shorten this on mobile. On desktop, it's nice how the top and bottom horizontal edges of the text area line up with the top and bottom edges of the image (so that extra space is good on desktop, but not needed on mobile).

I am also thinking that the error page should not overwrite the style of the nav and footer. Would be better if we can pass a variable into the nav/footer includes whether they should show the black or white themes. That makes it more self-contained and would allow for the themes to be used on other pages as well. What do you think?

404-page-feedback-201029

**Comment by mrsellars** [comment]
@ConorOkus @GBKS great, thanks for your feedback. I'll do the adjustments in the next days.

**Comment by GBKS** [comment]
@mrsellars let me know if you're still planning to work on this. It's just a few tweaks, I can also wrap it up if you're busy buying christmas presents or otherwise.

**Comment by mrsellars** [comment]
@GBKS I have some time at the weekend and will finish it then.

**Review by GBKS — APPROVED** [review]
tACK. Just made those tweaks I complained about on October 29. Tested, looks good, ready to go.

**Comment by ConorOkus** [comment]
Thanks also tested, looks good!

**Review by ConorOkus — APPROVED** [review]

**Comment by mrsellars** [comment]
Very cool! Thank you @GBKS and @ConorOkus ❤️.

**Pull request merged**